docs/conf.py

```diff
@@ -156,8 +156,18 @@ def setup(app):
 # In testing the 'html' target, this took the build time on a fast system from
 # 40s to 9s.
 
+# NOTE: When this is enabled, expect errors from sphix-build in the form of:
+#
+# (Sphinx 1.7:)
+#     WARNING: toctree contains reference to nonexisting document u'security_conop/index'
+#
+# (Sphinx 1.8:)
```

Comment:
This info may need to be documented in Rake task as well.  Unclear if anyone is going to look at the README.